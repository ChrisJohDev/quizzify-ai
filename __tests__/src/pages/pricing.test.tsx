import {render, screen }from '@testing-library/react';
import Pricing from '@/pages/pricing';
import '@testing-library/jest-dom'

describe('Pricing - snapshot test', () => {
  it('renders correctly', () => {
    const { container } = render(<Pricing />);
    expect(container).toMatchSnapshot();
  });
});