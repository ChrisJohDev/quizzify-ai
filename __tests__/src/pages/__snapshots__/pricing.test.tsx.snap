// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pricing - snapshot test renders correctly 1`] = `
<div>
  <div
    class="container flex flex-col mx-auto pricingPage"
  >
    <h1
      class="text-center"
    >
      Pricing Page
    </h1>
    <div
      class="mx-auto tableWrapper"
    >
      <table>
        <thead>
          <tr>
            <th>
              Plan
            </th>
            <th>
              Price
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              Basic
            </td>
            <td>
              $10/month
            </td>
          </tr>
          <tr>
            <td>
              Pro
            </td>
            <td>
              25$/month
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</div>
`;
